mirrulations-work-server/src/mirrserver/work_server.py

```diff
@@ -156,4 +156,8 @@ def _get_client_id():
     while not is_redis_available(redis_database):
         print("Redis database is busy loading")
         time.sleep(30)
-    redis_database.app.run(host='0.0.0.0', port=8080, debug=False)
+    server = create_server(redis_database)
+    if server is None:
```

Comment:
This check may be redundant as the is_redis_available() will wait until the redis database is loaded, but if there is some other sort of issue where the database is None, this will tell us there is an issue with Redis.